### ISSUE TITLE
building with `cuda` feature hits `rblas` resolve error

### ISSUE THREAD

**Issue by barzamin** [spearow/coaster-blas#5]
When building `juice-examples` with `cargo build --no-default-features --features cuda`, I get an error
```rust
error[E0433]: failed to resolve. Maybe a missing `extern crate rblas;`?
  --> /home/erin/.cargo/git/checkouts/coaster-blas-c027954122df0449/2a8f63c/src/transpose.rs:20:33
   |
20 |     pub fn to_rblas(&self) -> ::rblas::attribute::Transpose {
   |                                 ^^^^^ Maybe a missing `extern crate rblas;`?
// etc ... //
```

I traced this back to the `native` feature being disabled when I pass these options - and thus [this](https://github.com/spearow/coaster-blas/blob/2a8f63cd2560aac2fefc1c8e39fab8384a9de8ef/src/lib.rs#L56) line being compiled out - but [this](https://github.com/spearow/coaster-blas/blob/2a8f63cd2560aac2fefc1c8e39fab8384a9de8ef/src/transpose.rs) function still trying to reference `::rblas`.

**Comment by drahnr** [comment]
Nice catch, that should be a conditional to native.

Keep in mind that it is pretty uncommon to compile without `native` feature flag.

**Comment by barzamin** [comment]
Juice's README recommends building with `--no-default-features --features cuda`, so I'd expect that to work and not do this :)

**Comment by drahnr** [comment]
True, tbh the README needs some additional attention. Both are supposedly easy fixes and I will fix them within the next few days.
Thanks again for reporting!

**Comment by drahnr** [comment]
Well it seems that issue were not as simple as expected, there were a couple of mistakes introduced which prevented `FlatBox` from even existing since the whole `native` mod was unused, which just pops up after solving your mentioned issue. They should be all resolved by now.
Please try again with the latest git, I have yet to publish releases.
